test/config/TestConfig.cpp

```diff
@@ -33,6 +33,12 @@ TEST(Config, AllFeaturesEnabled)
     "device-defender": {
         "enabled": true,
         "interval": 300
+    },
+    "fleet-provisioning": {
+        "enabled": true,
+        "template-name": "template-name",
+		"csr-file": "csr-file",
```

Comment:
It looks like the formatting here is a bit weird, csr-file and device-key should probably be aligned with enabled and template-name

Reply:
Looks like an tabs vs space issue (looks perfect in text editor). The change is required in every test case and will have a separate PR to resolve the issue.